admin/config.yml

```diff
@@ -49,8 +52,59 @@ collections:
         file: "_data/team-alpha.json"
         description: "Team Alpha Members List"
         fields:
-          - {label: "Name", name: "name", widget: "string"}
-          - {label: "Rule", name: "rule", widget: "string"}
-          - {label: "Song", name: "song", widget: "string"}
-          - {label: "Person", name: "person", widget: "string"}
-          - {label: "Fact", name: "fact", widget: "string"}
+          - label: Entries
+            name: Entries
+            widget: list
+            fields:
+              - {label: Name, name: name, widget: string}
+              - {label: Rule, name: rule, widget: string}
+              - {label: Song, name: song, widget: string}
+              - {label: Person, name: person, widget: string}
+              - {label: Fact, name: fact, widget: string}
+              - {label: Team, name: team, widget: string}
+      - name: "alumni-bootcamp"
+        label: "Alumni Bootcamp"
+        file: "_data/alumni-bootcamp.yml"
+        description: "Bootcamp Alumni List"
+        fields:
+          - label: Entries
+            name: Entries
+            widget: list
+            fields:
+              - {label: Name, name: name, widget: string}
+              - {label: Website, name: website, widget: string}
+              - {label: Github, name: github, widget: string}
+              - {label: Linkedin, name: linkedin, widget: string}
+              - {label: Image, name: image, widget: string}
+              - {label: Position, name: position, widget: string}
+              - {label: Cohort, name: cohort, widget: string}
+      - name: "events"
+        label: "Events"
+        file: "_data/events.yml"
+        description: "Current Events"
+        fields:
+          - label: Entries
+            name: Entries
+            widget: list
+            fields:
+              - {label: title, name: title, widget: string}
+              - {label: date, name: date, widget: string}
+              - {label: eventbrite_url, name: eventbrite_url, widget: string}
+              - {label: feature, name: feature, widget: boolean, default: false}
+              - {label: location, name: location, widget: string}
+              - {label: image_url, name: image_url, widget: string}
+              - {label: id, name: id, widget: string, required: false}
+
+  - name: "Course Charts"
```

Comment:
@mhisle22 Let's make these a collection as well. We can talk more in detail, but basically anything that _might_ be its own page eventually, should be a collection. 

I am also am having troubles editing this one in the dashboard. Could you get it to work?